### PR TITLE
Add line groups to proto files

### PR DESCRIPTION
**Pull request by eturck**
This pull request is a sub-part of a bigger one in the navitia repository.
It needs to be merged before since the new code refer to the modified proto files.

I'm quite new to protobuf files, so if there is any mistake let me know.

### CODE REVIEW

File: type.proto [hunk 1]
```diff
@@ -216,6 +227,14 @@ message Line {
     optional uint32 closing_time = 19;
 
     repeated Property properties = 21;
+    repeated GroupLink groups = 23;
+}
+
+message LineGroup {
+    optional string name = 3;
```
Comment (kinnou02) [inline type.proto]: You can start at one, a lot of our messages start a three probably because we had remove some old fields.

File: type.proto [hunk 2]
```diff
@@ -196,7 +197,17 @@ message MultiLineString {
     repeated LineString lines = 1;
     optional double length = 2;
 }
-	
+
+message GroupLink {
+    optional bool is_main_line = 1;
```
Comment (antoine-de) [inline type.proto]: why do you need 2 is_main_line attributes ? I think one would be enough

**Comment by eturck** [comment]
The goal was to be able to get all the groups for a line, and all the lines for a group. (There is some example in the other PR https://github.com/CanalTP/navitia/pull/1071).
There is two different messages : 
* GroupLink : for a line, we are displaying each group it is part of, and we specify if it's the main_line of the group or not,
* LineLink : for a group, we are displaying all the line of the group, and for each one, we specify if it's the main_line.

Maybe it's not a good way to do it ? I'm not sure if I can achieve this with only one "is_main_line" attribute since it's in two different messages ?

**Comment by antoine-de** [comment]
fine for me, thx!

**Pull request merged**